### PR TITLE
Disable validation for subquery to prevent error

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

If there are more than 100 targets, the validation throws an error. This change skips the validation so no error is thrown.

Fixes https://github.com/appwrite/appwrite/issues/8299

## Test Plan

Added new test case which should pass

## Related PRs and Issues

* https://github.com/appwrite/appwrite/issues/8299

Parent PR:

* https://github.com/appwrite/appwrite/pull/8317

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/init.php [hunk 1]
```diff
@@ -610,9 +610,17 @@ function (mixed $value, Document $document, Database $database) {
...
+                $database->disableValidation();
+                $targets = $database->find('targets', [
+                    Query::equal('$internalId', $targetIds)
+                ]);
+                return $targets;
+            } catch (\Throwable $th) {
+                return [];
+            } finally {
+                $database->enableValidation();
+            }
```
Comment (abnegate) [inline app/init.php:623]: I don't think we should try/catch here, it will be hard to debug if something goes wrong

Comment (stnguyen90) [reply app/init.php]: Updated

File: app/init.php [hunk 2]
```diff
@@ -610,9 +610,15 @@ function (mixed $value, Document $document, Database $database) {
             ])
         ));
         if (\count($targetIds) > 0) {
-            return $database->find('targets', [
-                Query::equal('$internalId', $targetIds)
-            ]);
+            try {
+                $database->disableValidation();
```
Comment (abnegate) [inline app/init.php:614]: Let's use the new skip here

Comment (stnguyen90) [reply app/init.php]: Updated

**Review by abnegate — APPROVED** [review]

**Pull request merged**